confd/templates/makefile/makefile.tmpl

```diff
@@ -102,7 +102,7 @@ compile-common: check-java-version
 
 .PHONY: unit
 unit: update-props
-	cd src-java && ./gradlew test --stacktrace
+	cd src-java && ./gradlew checkstyleMain checkstyleTest test --stacktrace
```

Comment:
I'd prefer a dedicated target (make's target) to run checkstyle. Similar to the sonar target. This will keep flexibility, and not slow down the unit target.

Reply:
as you wish, updated